openupgradelib/openupgrade_120.py

```diff
@@ -225,25 +225,25 @@
     _r(selector=".bg-black-25", class_add="text-white"),
 
     # Image floating snippet disappears
-    _r(selector=".o_image_floating.o_margin_s.float-left", class_add="ml8"),
-    _r(selector=".o_image_floating.o_margin_s.float-right", class_add="mr8"),
+    _r(selector=".o_image_floating.o_margin_s.float-left", class_add="mr8"),
+    _r(selector=".o_image_floating.o_margin_s.float-right", class_add="ml8"),
     _r(selector=".o_image_floating.o_margin_m.float-left",
-       style_add={"margin-left": "12px"}),
-    _r(selector=".o_image_floating.o_margin_m.float-right",
        style_add={"margin-right": "12px"}),
-    _r(selector=".o_image_floating.o_margin_l.float-left", class_add="ml16"),
-    _r(selector=".o_image_floating.o_margin_l.float-right", class_add="mr16"),
-    _r(selector=".o_image_floating.o_margin_xl.float-left", class_add="ml32"),
-    _r(selector=".o_image_floating.o_margin_xl.float-right", class_add="mr32"),
+    _r(selector=".o_image_floating.o_margin_m.float-right",
+       style_add={"margin-left": "12px"}),
+    _r(selector=".o_image_floating.o_margin_l.float-left", class_add="mr16"),
+    _r(selector=".o_image_floating.o_margin_l.float-right", class_add="ml16"),
+    _r(selector=".o_image_floating.o_margin_xl.float-left", class_add="mr32"),
+    _r(selector=".o_image_floating.o_margin_xl.float-right", class_add="ml32"),
     _r(selector=".o_image_floating.o_margin_s", class_add="mb4"),
     _r(selector=".o_image_floating.o_margin_m", class_add="mb8"),
     _r(selector=".o_image_floating.o_margin_l",
        style_add={"margin-bottom": "12px"}),
     _r(selector=".o_image_floating.o_margin_xl", class_add="mb24"),
     _r(selector=".o_image_floating .o_footer", class_rm="o_footer"),
-    _r("s_image_floating", "col-5"),
+    _r(class_rm="s_image_floating"),
     _r("o_image_floating o_margin_s o_margin_m o_margin_l o_margin_xl",
-       "col-5", selector=".o_image_floating"),
+       "col-5 p-0", selector=".o_image_floating"),
```

Comment:
This is the expected behaviour?  Shouldn't be removed as with 's_image_floating'?

Reply:
No, this one has related styles that have to be replaced: https://github.com/odoo/odoo/blob/11.0/addons/website/static/src/less/website.snippets.less#L88-L90